ndcube/ndcube.py

```diff
@@ -22,15 +22,15 @@
 from astropy.wcs.wcsapi import BaseHighLevelWCS, HighLevelWCSWrapper
 
 from ndcube import utils
-from ndcube.extra_coords import ExtraCoords, ExtraCoordsABC
+from ndcube.extra_coords.extra_coords import ExtraCoords, ExtraCoordsABC
```

Comment:
Why not just `from ndcube import ExtraCoords, ExtraCoordsABC`?

Reply:
I personally subscribe to the idea that internal code should not call the public API. 

BUt I can make this change.